packages/teleterm/src/ui/TabHost/TabHost.test.tsx

```diff
@@ -0,0 +1,148 @@
+import { fireEvent, render } from 'design/utils/testing';
+import React from 'react';
+import { TabHost } from 'teleterm/ui/TabHost/TabHost';
+import { MockAppContextProvider } from 'teleterm/ui/fixtures/MockAppContextProvider';
+import { Document, DocumentsService } from 'teleterm/ui/services/docs';
+import { KeyboardShortcutsService } from 'teleterm/ui/services/keyboardShortcuts';
+import {
+  MainProcessClient,
+  TabContextMenuOptions,
+} from 'teleterm/mainProcess/types';
+import { getMockDocuments } from 'teleterm/ui/fixtures/mocks';
+import { ClustersService } from 'teleterm/ui/services/clusters';
+
+function getTestSetup() {
+  const mockedDocuments = getMockDocuments();
+
+  const keyboardShortcutsService: Partial<KeyboardShortcutsService> = {
+    subscribeToEvents() {},
+    unsubscribeFromEvents() {},
+  };
+
+  const mainProcessClient: Partial<MainProcessClient> = {
+    openTabContextMenu: jest.fn(),
+  };
+
+  const docsService: Partial<DocumentsService> = {
+    getDocuments(): Document[] {
+      return mockedDocuments;
+    },
+    getActive() {
+      return mockedDocuments[0];
+    },
+    close: jest.fn(),
+    open: jest.fn(),
+    closeOthers: jest.fn(),
+    closeToRight: jest.fn(),
+    openNewTerminal: jest.fn(),
+    swapPosition: jest.fn(),
+    duplicatePtyAndActivate: jest.fn(),
+    useState() {
+      return null;
+    },
+  };
+
+  const clustersService: Partial<ClustersService> = {
+    subscribe: jest.fn(),
+    unsubscribe: jest.fn(),
+    findRootClusterByResource: jest.fn(),
+    findCluster: jest.fn(),
+    findGateway: jest.fn()
+  };
+
+  const utils = render(
+    <MockAppContextProvider
+      appContext={{
+        // @ts-expect-error - using mocks
+        keyboardShortcutsService,
+        // @ts-expect-error - using mocks
+        docsService,
+        // @ts-expect-error - using mocks
+        mainProcessClient,
+        // @ts-expect-error - using mocks
+        clustersService,
+      }}
+    >
+      <TabHost />
+    </MockAppContextProvider>
+  );
+
+  return {
+    ...utils,
+    docsService,
+    mainProcessClient,
+  };
+}
+
+test('should render documents without home document', () => {
```

Comment:
Do we use any convention when it comes to how tests are supposed to be named? I quickly grepped through the project and it seems that tests starting with "should" are uncommon and most of them start with a more direct form.

```
$ rg "^test" | wc -l
     288
$ rg "^test\('should" | wc -l
      14
```

Perhaps it would make sense to rephrase them? Just dropping the "should" should accomplish that, I think.

Reply:
Thanks, done